tools/wikipathways/wikipathways_fetch.xml

```diff
@@ -0,0 +1,25 @@
+<tool id="wikipathways_fetch" name="WikiPathways fetch data" version="1.0">
+    <description>WikiPathways fetch data </description>
+    <requirements>
+        <requirement type="package" version="1.7.1">r-jsonlite</requirement>
+        <requirement type="package" version="1.6.6">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+Rscript '$__tool_directory__/wikipathways_fetch.R'
+-o '${outputJSON}'
+    ]]></command>
+    <outputs>
+        <data name="outputJSON" format="json" label="${tool.name} on ${on_string}: JSON" />
```

Comment:
`on_string` is empty if there are no inputs, probably better to just describe the contents of the file

Reply:
Yes, you're right. Will change that 👍

Reply:
I changed it, but now it gives errors???